.gitignore

```diff
@@ -1,4 +1,4 @@
 .idea/
 .vscode/
-
+food/
```

Comment:
I think this should be grouped differently in paragraph.

Reply:
Solved